datadog/resource_datadog_synthetics_test_.go

```diff
@@ -197,6 +197,52 @@ func syntheticsTestOptions() *schema.Schema {
 					Type:     schema.TypeBool,
 					Optional: true,
 				},
+				"monitor_options": syntheticsTestMonitorOptions(),
+			},
+		},
+	}
+}
+
+func syntheticsTestMonitorOptions() *schema.Schema {
+	return &schema.Schema{
+		Type: schema.TypeMap,
+		DiffSuppressFunc: func(key, old, new string, d *schema.ResourceData) bool {
+			if key == "options.follow_redirects" || key == "options.accept_self_signed" {
```

Comment:
Hmm, this seems like a copy-paste error. You probably wanted to look at `renotify_interval` and convert it to `int`?

Reply:
@bkabrda you have good 👀 
```suggestion
			if key == "options.monitor_options.renotify_interval" {
```